fe/fe-core/src/main/java/org/apache/doris/qe/DdlExecutor.java

```diff
@@ -230,8 +229,6 @@ public static void execute(Env env, DdlStmt ddlStmt) throws Exception {
             env.getBackupHandler().createRepository((CreateRepositoryStmt) ddlStmt);
         } else if (ddlStmt instanceof DropRepositoryStmt) {
             env.getBackupHandler().dropRepository((DropRepositoryStmt) ddlStmt);
-        } else if (ddlStmt instanceof SyncStmt) {
-            return;
```

Comment:
why remove this?

Reply:
Move to here https://github.com/apache/doris/blob/b4b8cdb803f11f40111e3bc95275dc61a5d4531f/fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java#L728-L729